indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -252,6 +255,37 @@ def _process(self):
                                                     regform=self.regform)
 
 
+class RHRegistrationFormNotificationPreview(RHManageRegFormBase):
+    """Preview registration emails."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.commit = False
```

Comment:
We usually do not override `__init__` of RHs; I'd just set this in `_process`

Reply:
Done 👍